includes/options/class-amp-options-manager.php

```diff
@@ -327,6 +328,45 @@ public static function update_analytics_options( $data ) {
 		return self::update_option( 'analytics', wp_unslash( $data ) );
 	}
 
+	/**
+	 * Renders the welcome notice on the 'AMP Settings' page.
+	 *
+	 * Uses the user meta values for the dismissed WP pointers.
+	 * So once the user dismisses this notice, it will never appear again.
+	 *
+	 * @todo: Add the full copy for this notice when it is decided.
+	 */
+	public static function render_welcome_notice() {
+		if ( 'toplevel_page_' . self::OPTION_NAME !== get_current_screen()->id ) {
+			return;
+		}
+
+		$notice_id = 'amp-welcome-notice-1';
+		$dismissed = get_user_meta( get_current_user_id(), 'dismissed_wp_pointers', true );
+		if ( in_array( $notice_id, explode( ',', strval( $dismissed ) ), true ) ) {
+			return;
+		}
+
+		?>
+		<div class="notice notice-info is-dismissible" id="<?php echo esc_attr( $notice_id ); ?>"><p><?php esc_html_e( 'Welcome to AMP for WordPress', 'amp' ); ?></p></div>
+
+		<script>
+		( function( $ ) {
+			$( function() {
+				// On dismissing the notice, make a POST request to store this notice with the dismissed WP pointers so it doesn't display again.
+				$( document ).on( 'click', <?php echo wp_json_encode( "#$notice_id .notice-dismiss" ); ?>, function() {
```

Comment:
Another point, this code is using event delegation when it doesn't need to. It should instead just be:

```
$( <?php echo wp_json_encode( "#$notice_id .notice-dismiss" ); ?> ).on( 'click', function() {...});
```

Reply:
Hi Weston,
Good point. 

But it looks like the `button.notice-dismiss` might be created with JS when the `is-dismissible` class is present, and might not yet exist at the time the `<script>` above runs. 

```html
<button type="button" class="notice-dismiss">
```

So the `click` handler with that selector doesn't run.

Maybe if this script ran later, the `button.notice-dismiss` would be present.

Reply:
ok, that makes sense

Reply:
At least we can do this:
```js
$( <?php echo wp_json_encode( "#$notice_id" ); ?> ).on( 'click', '.notice-dismiss', function() {
```
That will scope the delegated event to the containing element.